apps/heft/src/utilities/TaskPackageResolver.ts

```diff
@@ -91,23 +114,27 @@ export class TaskPackageResolver {
     };
   }
 
-  private static _tryResolveTaskPackage(
+  private _tryResolveTaskPackage(
     taskPackageName: string,
-    projectFolder: string,
+    tsconfigBaseWithTypescriptDependencyFolder: string,
     rigPackageFolder: string | undefined,
     terminal: Terminal
   ): string | undefined {
-    let result: string | undefined = undefined;
-    if (rigPackageFolder) {
-      result = TaskPackageResolver._tryResolvePackage(taskPackageName, rigPackageFolder, terminal, true);
-    }
-    if (!result) {
-      result = TaskPackageResolver._tryResolvePackage(taskPackageName, projectFolder, terminal, false);
+    let result: string | undefined = this._tryResolvePackage(
+      taskPackageName,
+      tsconfigBaseWithTypescriptDependencyFolder,
+      terminal,
+      false
+    );
+
+    if (!result && rigPackageFolder) {
+      result = this._tryResolvePackage(taskPackageName, rigPackageFolder, terminal, true);
     }
+
     return result;
   }
 
-  private static _tryResolvePackage(
+  private _tryResolvePackage(
```

Comment:
This method could also be made async so that you're async through the entire i/o stack... though that could easily be "out of scope" so I'm not picky

Reply:
`Import.resolvePackage` doesn't have an async variant yet, unfortunately.